packages/server/lib/project-base.ts

```diff
@@ -160,19 +160,16 @@ export class ProjectBase extends EE {
 
     this._server = new ServerBase(cfg)
 
-    let studioManager: StudioManager | null
-
-    if (process.env.CYPRESS_ENABLE_CLOUD_STUDIO || process.env.CYPRESS_LOCAL_STUDIO_PATH) {
-      studioManager = await getAndInitializeStudioManager({
-        projectId: cfg.projectId,
-        cloudDataSource: this.ctx.cloud,
-      })
-
+    // Start studio manager initialization but don't block
+    this.studioManagerPromise = getAndInitializeStudioManager({
```

Comment:
Wondering if it might be better to have this promise live on ctx.coreData and maybe return the studio manager. So maybe coreData would look like:

- studioManagerPromise
- getStudio - replaces `studio` and instead is an async function that awaits the studioManager promise and returns it.

That way the places that use `studio` don't have to remember to await the promise. That's just how you access it. Thoughts?